build_docs.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
+# To build manually run "sudo sh build_docs.sh"
 gem install jazzy
 echo -e "Copying MSAL public files"
-mkdir docs.temp
-mkdir docs.temp/MSAL
+mkdir -m 777 docs.temp
```

Comment:
Why did we change the permission of this file? Shouldn't we use the default permission instead of 777?

Reply:
Sorry, I should have copied over this comments from ADO:
While running locally, the 777 permission needs to be added otherwise sourcekitten command fails because it cannot write into the json's. It could be on server side the creation of the folder works differently as the terminal might be started with different parameters. The 777 permission gives access to all users or scripts to write into the folder/files in folder.

Reply:
Can you please check if we need this permission also in the server side?

Reply:
It doesn't need to be added so I removed it. Will check before deployment to make sure script runs correctly.